src/scenes/Moves/Ppm/DocumentsUploaded.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import Alert from 'shared/Alert';
+import { selectPPMCloseoutDocumentsForMove } from 'shared/Entities/modules/movingExpenseDocuments';
+import { getMoveDocumentsForMove } from 'shared/Entities/modules/moveDocuments';
+import { connect } from 'react-redux';
+
+export class DocumentsUploaded extends Component {
+  componentDidMount() {
+    const { moveId } = this.props;
+    this.props.getMoveDocumentsForMove(moveId);
+  }
+
+  createHeaderMessage = documentLength => {
+    return (
+      <p>
```

Comment:
```suggestion
      <div style={{ marginBottom: 5, paddingTop: 5 }}>
```

Changing the `p` tag to a `div` should fix the alignment issues and the margin/padding additions to the div should have it more centered

Reply:
🎉 🎉 🎉

Reply:
looks better. updated screenshots above.